cmd/otel-allocator/allocation/consistent_hashing.go

```diff
@@ -22,13 +22,14 @@ import (
 	"github.com/go-logr/logr"
 	"github.com/prometheus/client_golang/prometheus"
 
+	"github.com/open-telemetry/opentelemetry-operator/apis/v1alpha1"
 	"github.com/open-telemetry/opentelemetry-operator/cmd/otel-allocator/diff"
 	"github.com/open-telemetry/opentelemetry-operator/cmd/otel-allocator/target"
 )
 
 var _ Allocator = &consistentHashingAllocator{}
 
-const consistentHashingStrategyName = "consistent-hashing"
+const consistentHashingStrategyName = string(v1alpha1.OpenTelemetryTargetAllocatorAllocationStrategyConsistentHashing)
```

Comment:
question: are we okay having the allocator import this from the operator? It means that the allocator now has a required dependency on the operator which i'm not sure we want. cc @pavolloffay

Reply:
I think it is fine here, as mentioned in the description we are not adding any additional dependencies.  
The benefit would be that we have source of truth regarding the available options.